stl/inc/yvals_core.h

```diff
@@ -805,8 +806,8 @@
 // clang-format off
 #define _STL_DISABLED_WARNINGS                        \
     4180 4412 4455 4494 4514 4574 4582 4583 4587 4588 \
-    4619 4623 4625 4626 4643 4648 4702 4793 4820 4988 \
-    5026 5027 5045 5220 6294                          \
+    4619 4623 4625 4626 4643 4648 4702 4793 4820 4868 \

```

Comment:
There are 3 suppressions in `<execution>` which can now be removed, further reinforcing the goodness of this change.